makefile

```diff
@@ -52,8 +58,17 @@ vendor:
 	@glide cc
 	@glide update
 
-test:
-	@$(BUILD_ENV) go test $(BUILD_FLAGS) -cover -v -timeout 10s -race $$(go list ./...|grep -v vendor)
+unit:
+	@echo "go tests SDK"
+	@$(BUILD_FLAGS) go test $(BUILD_FLAGS) -v -tags ${UNIT_TEST_TAGS} $(UNIT_TEST_ONLY_PKGS)
+
+unit-with-race-cover:
+	@echo "go test SDK witch race and cover"
+	@$(BUILD_FLAGS) go test $(BUILD_FLAGS) -cover -timeout 10s -race -tags ${UNIT_TEST_TAGS} $(UNIT_TEST_ONLY_PKGS)
```

Comment:
Any reason to separate this?
race and timeout should always be enabled, cover is debateable but does not really have a negative impact (afaik).